models/Profile.js

```diff
@@ -23,6 +23,9 @@ const ProfileSchema = new mongoose.Schema(
       lat: Number,
       lon: Number,
       updatedAt: Date,
+      city: String,
```

Comment:
There is no reason to modify the database model as you are not writing to the database.

Reply:
But according the discussion,  extra fields including city, state and country were suggested to be added in the profile. That's why I modified the schema.

Reply:
While this may have been the original idea, #3169 implemented location by pulling the data from GitHub, then obtaining the coordinates & full name from a mapping system.

Changing this would require more discussion outside the scope of using the existing location data for search.